plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -20,18 +20,27 @@ export class KafkaQueue {
     public pluginsServer: Hub
     public workerMethods: WorkerMethods
     public consumerReady: boolean
+    private lastHeartbeat: number | undefined
     private kafka: Kafka
     private consumer: Consumer
+    private consumerConfig: Partial<ConsumerConfig>
     private consumerGroupMemberId: string | null
     private wasConsumerRan: boolean
     private ingestionTopic: string
     private eventsTopic: string
     private eachBatch: Record<string, EachBatchFunction>
 
-    constructor(pluginsServer: Hub, workerMethods: WorkerMethods) {
+    constructor(pluginsServer: Hub, workerMethods: WorkerMethods, consumerConfig: { sessionTimeout: 30000 }) {
```

Comment:
This doesn't seem correct - you're passing `{ sessionTimeout: 30000 }` as a type and we're not overriding this anywhere.

Please put plugin-server configuration into plugin-server/src/config/config.ts and use that.